src/components/login/LogInForm.js

```diff
@@ -29,6 +30,10 @@ const LogInForm = () => {
 
   const onSubmit = async (data) => {
     console.log(data);
+    const loginResult = await login(data);
+    if (loginResult.data.status === 'Success') {
+      await me();
```

Comment:
```suggestion
      const meChk = await me();
      if (meChk.status !== 'Success') {
        alert("에러 발생했어용~ 페이지 쿠키 지우고 강제 리프레쉬 부탁드립니다");
      }
```

이런 식으로 me fail 에 대한 처리가 필요해보여요!